src-db/database/sourcedata/AD_MESSAGE.xml

```diff
@@ -17956,6 +17956,18 @@ Do you want to enable the Heartbeat and continue?]]></MSGTEXT>
 <!--2EB6F2EE079E41C3A9E416768353105F-->  <ISINCLUDEINI18N><![CDATA[N]]></ISINCLUDEINI18N>
 <!--2EB6F2EE079E41C3A9E416768353105F--></AD_MESSAGE>
 
+<!--2F0E4BF492614F0D96E651BDA5DCE968--><AD_MESSAGE>
+<!--2F0E4BF492614F0D96E651BDA5DCE968-->  <AD_MESSAGE_ID><![CDATA[2F0E4BF492614F0D96E651BDA5DCE968]]></AD_MESSAGE_ID>
+<!--2F0E4BF492614F0D96E651BDA5DCE968-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
+<!--2F0E4BF492614F0D96E651BDA5DCE968-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
+<!--2F0E4BF492614F0D96E651BDA5DCE968-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--2F0E4BF492614F0D96E651BDA5DCE968-->  <VALUE><![CDATA[AD_Unique_Check]]></VALUE>
+<!--2F0E4BF492614F0D96E651BDA5DCE968-->  <MSGTEXT><![CDATA[A sequence with the same AD_Table_ID, AD_Column_ID and C_Doctype_ID already exists.]]></MSGTEXT>
```

Comment:
This may be "A sequence with the same table, column and document type already exists.". It is more user friendly.

Reply:
That's a good point, thank you!